page/style.css

```diff
@@ -1,223 +1,607 @@
+
+.dashedborderbottom {
+	margin-top: 99px;
+	border-bottom: 3px dashed #234f70;
+}
+
+.rottentomato {
+	width: 100%;
+	border-top: 5px solid #3476a8;
+	border-bottom: 5px solid #3476a8;
+	float: left;
+}
+
+.rottentomato > p:first-child {
+	margin: 8px 5px 5px 12px;
+	float: left;
+	font-size: 12.2px;
+	line-height: 23px;
+	color: #75a8d0;
+}
+
+.rottenrating {
+	font-size: 27.2px;
+}
+
+.rottenpercentage {
+	float: left;
+	margin: 0;
+	font-size: 44px;
+	color: white;
+}
+
+.rottenratingbar {
+	margin-left: 15px;
+	margin-bottom: 10px;
+	padding: 2px;
+	margin-top: 70px;
+	width: 80%;
+	height: 5px;
+	background-color: #619cc9;
+}
+
+.rottenratingbar > span {
+	display: block;
+	width: 40%;
+	height: 130%;
+	background-color: white;
+}
+
+.barworldpremier {
+	display: -webkit-box;
+	display: -moz-box;
+	display: -ms-flexbox;
+	display: -webkit-flex;
+	display: flex;
+	margin-left: 16px;
+	margin-top: 12px;
+	height: 10px;
+}
+
+.barworldpremier p {
+	padding-left: 8px;
+	padding-right: 8px;
+	color: #75a8d0;
+	margin: 0px;
+	font-size: 12px;
+}
+
+.barworld {
+	padding: 4px;
+	width: 15%;
+	height: 5px;
+	background-color: #619cc9;
+}
+
+.worldpremiere > p {
+	margin: 0px;
+	font-family: oswald;
+	color: white;
+	font-size: 22px;
+	height: 22px;
+}
+
+.worldpremiere > p:nth-child(odd) {
+	color: #75a8d0;
+	margin: 0px;
+	height: 22px;
+}
+
+.circlewhite {
+	margin-top: 10px;
+	margin-left: 78px;
+	background: white;
+	width: 40px;
+	height: 40px;
+	border-radius: 50%;
+}
+
+.overflow {
+	width: 730px;
+	overflow: visible;
+	margin-left: 55px;
+	margin-top: -66px;
+	z-index: 2;
+	padding-bottom: 35px;
+}
+
+.overflow > figure {
+	height: 140px;
+}
+
+.overflow > figure > img {
+	height: 300px;
+}
+
+.overflow > p {
+	font-size: 13px;
+	font-family: lora;
+	font-style: italic;
+	color: #5e6c74;
+	width: 550px;
+}
+
+.overflow-one {
+	transform: rotate(1.5deg);
+	text-align: right;
+	margin-left: 235px;
+	padding-top: 15px;
+	line-height: 1.5;
+}
+
+.overflow-two {
+	transform: rotate(-1.5deg);
+	text-align: left;
+	margin-left: 70px;
+	padding-top: 10px;
+	line-height: 1.5;
+}
+
+.overflow-three {
+	transform: rotate(1.5deg);
+	margin-left: 140px;
+	padding-top: 15px;
+	line-height: 1.5;
+}
+
+.overflow-one > span {
+	font-weight: 900;
+	color: black;
+}
+
+footer {
+	width: 730px;
+	display: inline-block;
+	margin-left: 110px;
+	border-top: 1px solid lightgray;
+}
+
+.footer-issue {
+	display: inline-block;
+	color: gray;
+	font-family: lato;
+	font-weight: 300;
+}
+
+a:-webkit-any-link {
+	color: gray;
+	font-family: lato;
+	font-weight: 300;
+	text-decoration: none;
+}
+
+.footer-address {
+	float: right;
+	text-align: right;
+	color: gray;
+	font-family: merriweather;
+	font-size: 12px;
+	font-weight: 100;
+}
+
+.footer-bold {
+	font-weight: 600;
+	color: gray;
+}
+
+form {
+	width: 236px;
+	position: fixed;
+	bottom: 12px;
+	right: 12px;
+	border: 1px;
+	border-radius: 10px;
+	background: -webkit-linear-gradient(left top, violet, #619cc9);
```

Comment:
No need for vendor prefixing ... I feel like you got this out of a code generator?